RELEASE-NOTES.md

```diff
@@ -9,6 +9,7 @@
 - Added `Matern12` covariance function for Gaussian processes. This is the Matern kernel with nu=1/2.
 - Progressbar reports number of divergences in real time, when available [#3547](https://github.com/pymc-devs/pymc3/pull/3547).
 - Sampling from variational approximation now allows for alternative trace backends [#3550].
+- Included `fast_sample_posterior_predictive`, which draws samples from the posterior predictive distribution taking the maximum advantage possible from operation vectorization. This will implementation is likely to run into problems with multivariate distributions for now.
```

Comment:
```suggestion
- Included `fast_sample_posterior_predictive`, which draws samples from the posterior predictive distribution taking the maximum advantage possible from vectorizating the operation. This implementation will likely run into problems with multivariate distributions currently.
```

Reply:
Made minor corrections to @junpenglao 's edit.  Should we add a phrase to explain *why* there are likely to be problems with multivariate distributions?  Is this a shape issue, or is this a problem with the way these distributions' `random` methods work, or both?